plugin/storage/es/spanstore/dbmodel/converter.go

```diff
@@ -0,0 +1,229 @@
+package dbmodel
+
+import (
+	"strconv"
+	"errors"
+	"encoding/hex"
+
+	"github.com/uber/jaeger/model"
+)
+
+var ErrUnknownKeyValueTypeFromES = errors.New("Unknown tag type found in ES")
+
+// FromDomain converts a domain model.Span to a database Span
+func FromDomain(span *model.Span) *Span {
+	return converter{}.fromDomain(span)
+}
+
+// ToDomain converts a database Span to a domain model.Span
+func ToDomain(dbSpan *Span) (*model.Span, error) {
```

Comment:
I modeled this after the cassandra converter. Splitting these into separate files would stray away from that consistency; is that okay?